contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -524,6 +524,12 @@ function Start-OpenSSHBuild
         $xml.Project.PropertyGroup.AdditionalDependentLibs = 'onecore.lib'
         $xml.Project.PropertyGroup.MinimalCoreWin = 'true'
         $xml.Save($PathTargets)
+        
+        #Use onecore libcrypto binaries
+        (Get-Content $PathTargets).Replace('LibreSSLSDK\x86','LibreSSLSDK\onecore\x86') | Set-Content $PathTargets
```

Comment:
to be consistent, this can be achieved by before line 526. 
$xml.Project.PropertyGroup.LibreSSL-x86-Path = 'LibreSSLSDK\onecore\x86'
$xml.Project.PropertyGroup.LibreSSL-x64-Path = 'LibreSSLSDK\onecore\x64'
$xml.Project.PropertyGroup.LibreSSL-arm64-Path = 'LibreSSLSDK\onecore\arm64'
$xml.Project.PropertyGroup.LibreSSL-arm-Path = 'LibreSSLSDK\onecore\arm'

Reply:
also is LibreSSL-Path still used? is there any need to update its value?